docs/reference/configuration/configuration.md

```diff
@@ -15,12 +15,13 @@ Here is how a default JSON file might look like:
         "cellular.random_max_start_delay": "100"
     },
     "K64F": {
-        "cellular.use-apn-lookup": false
+        "cellular.use-apn-lookup": false,
+        "platform.stdio-baud-rate": 9600
```

Comment:
Since 9600 is the default value for this target, may be worth changing to something like `115200` (no strong feelings one way or the other though).

Reply:
Hmm. I see your point. However changing it to something that is not valid for that target makes me bit nervous.

Reply:
Good point. Since this is configuring for the K64F specifically I'd think it would be fine but could be confusing if someone is using a different target and just copy/pasted this.

Reply:
For a parameter like baud rate there is only really one right answer, but I see your point. In that case, can you suggest an alternative parameter that is not so susceptible to change in targets. Alternatively, I can add a note which says that this parameter is for K64F and might be different for another target.

Reply:
Actually looking at the description we have for the snippet, it should be clear that the baud-rate value here should only be set for K64F.